state/immutable-trie/storage.go

```diff
@@ -36,6 +36,7 @@ type Storage interface {
 	Batch() Batch
 	SetCode(hash types.Hash, code []byte) error
 	GetCode(hash types.Hash) ([]byte, bool)
+	Has(k []byte) (bool, error)
```

Comment:
Maybe to use types.Hash as a param to be consistent with above methods Set/GetCode?